_run/ssh/.envrc

```diff
@@ -2,8 +2,11 @@ source_up .envrc
 
 dotenv_if_exists dev.env
 
-source_env ~/projects/akash/gpu
+AP_RUN_NAME=$(basename "$(pwd)")
+AP_RUN_DIR="${DEVCACHE_RUN}/${AP_RUN_NAME}"
 
-export AKASH_HOME=$DEVCACHE_RUN/ssh/.akash
+export AKASH_HOME="${AP_RUN_DIR}/.akash"
 export AKASH_KUBECONFIG=$KUBECONFIG
 export AP_KUBECONFIG=$KUBECONFIG
+export AP_RUN_NAME
+export AP_RUN_DIR
```

Comment:
`export KUBE_SSH_NODES="root@<YOUR_CLUSTER_IP>"` will still need to be manually added. CC @chainzero